content/source/docs/enterprise-beta/private/config.html.md

```diff
@@ -0,0 +1,242 @@
+account credentials you just created. Do so and you will be returned to the
+`Configure SMTP` step.
+
+### Configure SMTP
+
+Terraform Enterprise sends emails occasionally for features like password reset and to alert you
+when a new administrator is added. Fill out the details for your SMTP server.
+You may use an external provider such as Amazon SES or Mailgun.
+
+When you have added your configuration press `Save and Continue`, or skip to the
+next step.
+
+### Configure Twilio
+
+Terraform Enterprise uses Twilio to send SMS if a user selects SMS as their multi-factor
+authentication option. If Twilio is not configured they must use the Google
+Authenticator app on their smartphone instead.
+
+When you have added your configuration press `Save and Continue`, or skip to the
+next step.
+
+### Configure Git(Hub)
+
+To use most Private Terraform Enterprise features you will need to configure it to ingest data from git.
+This step is titled `Configure GitHub`, and the example steps assume GitHub, but
+you can select other git providers such as GitLab or Bitbucket from the dropdown.
+
+1. Fill in HTTP URL and API URL in Terraform Enterprise for your git provider
+
+	> Note to users of github.com - due to a bug in GitHub's OAuth
+	authentication, the HTTP URL must be set to https://github.com. This differs
+	from the example text, which includes a www.
+
+2. Create a new OAuth application in GitHub (for example,
+	<https://github.com/settings/applications/new>)
+
+3. Fill out the Homepage URL in GitHub to match your Private Terraform Enterprise URL
+
+4. Leave the Authorization callback URL on GitHub **blank** and click `Create
+	Application`
+
+5. Copy the Client ID and Client Secret from GitHub into the corresponding Terraform Enterprise
+	fields.
+
+6. Click `Save and Continue`.
+
+7. Copy the new callback URL from Terraform Enterprise into GitHub. (For example:
+	<https://tfe.example.com/auth/[uuid]/callback>) and click `Update
+	application` in GitHub.
+
+8. Click `Connect` and follow the GitHub OAuth flow.
+
+9. When you see "Successfully linked to GitHub" click `Logged in with GitHub` to
+	proceed to the next step.
+
+If you need to change a setting, click `Reconfigure` at the bottom of the page.
+If you have already completed this step or navigated away from the page, you can
+edit the OAuth configuration by way of the `here` link in the yellow bar at the
+top of the `Configure GitHub` page.
+
+### Configure Acceptance Tests, Verify, and Complete
+
+The `Configure Acceptance Tests`, `Configure Acceptance Test GitHub`, `Verify`,
+and `Complete` steps are no longer used and will be removed in a future version
+of Private Terraform Enterprise.
+
+After configuring your git provider, you are finished with the System
+Configuration section and should move on to Team Setup.
+
+## Team Setup
+
+Private Terraform Enterprise models Packer and Terraform projects as Builds and Environments,
```

Comment:
This seems like it comes from V1-land. Should it be here?

Reply:
@phinze I'm assuming we should drop 'packer' from there and just have it read 'Private Terraform Enterprise models projects as Builds and Environments...' ?